### PR TITLE
Add @scroll-timeline

### PR DESCRIPTION
**Pull request by argyleink**
Getting the ball rolling.

### CODE REVIEW

File: features-json/css-scroll-timeline.json [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+      "80":"n",
+      "81":"n",
+      "83":"n",
+      "84":"n",
+      "85":"y#2",
+      "86":"y#2",
+      "87":"y#2",
+      "88":"y#1",
+      "89":"n",
+      "90":"y"
```
Comment (flackr) [inline features-json/css-scroll-timeline.json:220]: 89 and 90 should be the same status as 88 until we have flipped the flag.

**Review by flackr — APPROVED** [review]
LGTM modulo fixing status in 89 and 90.

File: features-json/css-scroll-timeline.json (hunk 1 above)
Comment (argyleink) [inline features-json/css-scroll-timeline.json:220]:
```suggestion
      "90":"n"
```
cool, like this? set 90 to n?

Comment (flackr) [reply features-json/css-scroll-timeline.json:220]: I'm not sure I understand, why would 89 and 90 be no if 88 is y#1?

Comment (argyleink) [inline features-json/css-scroll-timeline.json:220]:
```suggestion
      "89":"y#1",
      "90":"y#1"
```

Comment (flackr) [reply features-json/css-scroll-timeline.json]: Looks good, thanks!

**Comment by foolip** [comment]
@argyleink asked if I have some tooling that would help with this PR, and the short answer is that http://mdn-bcd-collector.appspot.com/tests/api/CSSScrollTimelineRule is a test for this. But it sounds like this isn't enabled anywhere by default yet, and I can't find any docs on MDN or a BCD entry. Is someone working on that too? cc @jpmedley

**Comment by argyleink** [comment]
what can I do to help this merge / move forward? 🙂

**Comment by Fyrd** [comment]
@argyleink For support behind a flag the values should look like `n d #1` (also note the space separation). I'd also suggest testing in other Chromium-based browsers as I imagine support there will be similar. Thanks!

File: features-json/css-scroll-timeline.json [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+      "80":"n",
+      "81":"n",
+      "83":"n",
+      "84":"n",
+      "85":"y#2",
+      "86":"y#2",
+      "87":"y#2",
+      "88":"y#1",
+      "89":"y#1",
+      "90":"y#1"
```
Comment (argyleink) [inline features-json/css-scroll-timeline.json:220]:
```suggestion
      "85":"n d #2",
      "86":"n d #2",
      "87":"n d #2",
      "88":"n d #1",
      "89":"n d #1",
      "90":"n d #1"
```

**Review by argyleink — COMMENTED** [review]
I've double checked all other Chromium browsers for support (still none) 👍🏻 
I've updated the syntax as request 👍🏻 

@Fyrd how's it lookin now!?

**Comment by Fyrd** [comment]
Looking good, thank you! Have merged the PR. Will do further review shortly before it appears on the site.

**Comment by Fyrd** [comment]
Now available at https://caniuse.com/css-scroll-timeline

**Pull request merged**